### ISSUE TITLE
Chrome's high hardware consumption

### ISSUE THREAD

**Issue by amirhossein-fzl** [shamhi/TapSwapBot#162]
Chrome driver has high hardware consumption. Please switch to Firefox if possible. Thankful

**Comment by cgcore** [comment]
> Chrome driver has high hardware consumption. Please switch to Firefox if possible. Thankful

YES - my server consumption is high- and freez after run - before update it worked well

![image](https://github.com/shamhi/TapSwapBot/assets/106352513/aebadf8c-be06-4d24-9bcc-6468f0f30394)

**Comment by Ozzy4t** [comment]
> > Chrome driver has high hardware consumption. Please switch to Firefox if possible. Thankful
> 
> YES - my server consumption is high- and freez after run - before update it worked well
> 
> ![image](https://private-user-images.githubusercontent.com/106352513/346365028-aebadf8c-be06-4d24-9bcc-6468f0f30394.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA1NDMyMzUsIm5iZiI6MTcyMDU0MjkzNSwicGF0aCI6Ii8xMDYzNTI1MTMvMzQ2MzY1MDI4LWFlYmFkZjhjLWJlMDYtNGQyNC05YmNjLTY0NjhmMGYzMDM5NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOVQxNjM1MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mMDc2MTZmMjc2YjY0ZDJlMzk3MGQ3NjE0ZmQyYTFmNWExYTRmNzI2OGY2MzIyYzExMTI2ZTg2YmExYmMyNDdhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.JUPELIgIyZsFec9pNdyU0-ecS5iY7F5pZvjxDWX-Atk)

Does it also hang when creating a session? If so, try pressing the time synchronisation button in Windows
![image](https://github.com/shamhi/TapSwapBot/assets/172856987/b22d730d-ce41-4b3d-b8a9-db8f66e8a104)
